docs/tutorials/enterprise_beginner/3_using_existing_blocks.md

```diff
@@ -154,9 +150,9 @@ As you can see, the configuration at this point defines one view for the schedul
 
 See the `data-studio-slots-container` (line 5) and `data-studio-slot` (line 7) attributes? Without them you won't be able to place Content in the slots of the Schedule block, so don't forget about them if you decide to modify the template.
 
-When you look at the template, you can see three blocks, each of which will render the Content items using the `featured` view. As you may remember, so far we only have templates for `full` view for Articles. This means we need to create a `featured` view template for it as well, otherwise we will get an error when trying to add Content to the block.
+When you look at the template, you can see three blocks, each of which will render the Content items using the `featured` view. As you may remember, so far you only have templates for `full` view for Articles. This means you need to create a `featured` view template for it as well, otherwise you will get an error when trying to add Content to the block.
```

Comment:
Unrelated to your work here, however, this " you can see three blocks" calls for a screenshot.

Reply:
I believe it is in relation to modification made on `app/config/layouts.yml` described in paragraph above. Some blocks were added to the file and it was shown in the code so not sure if we need a screenshot.